luda-editor/luda-editor-client/src/app/editor/mod.rs

```diff
@@ -71,10 +71,7 @@ impl namui::Entity for Editor {
                             last_mouse_position_in_time: *click_in_time,
                         }));
                     }
-                    self.selected_clip_id = Some(clip_id.clone());
-                    self.clip_editor = Some(ClipEditor::new(
-                        &self.get_sequence().get_clip(clip_id).unwrap(),
-                    ));
+                    self.select_clip(clip_id);
```

Comment:
it's for managing selected clip behavior because undo would make selected_clip_id dangling. It's not very about this PR but I just fix the bug.

Think about this
1. you copy clip
2. select copied 
3. and undo. 
Then selected_clip_id become dangling. I tried to fix the problem about it.